main.go

```diff
@@ -22,7 +22,7 @@ func main() {
 		"detect a level from a given hash and home directory")
 	encrypt_flag := flag.Bool("enc", false, "encrypt a given challenge")
 	decrypt_flag := flag.Bool("dec", false, "decrypt a given challenge")
-	template_flag := flag.Bool("temp", false,
+	template_flag := flag.Bool("generate-from-template", false,
```

Comment:
@mrshu Could we perhaps add a short version of this argument something like `-gen` or just `-g` so there would be a long version that is self explanatory and short version that is just easier to write when used from command line?